mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -15,12 +18,17 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 	FIELD field_10980 ORIGIN Lnet/minecraft/class_2338;
 		COMMENT The block position which x, y, and z values are all zero.
 	FIELD field_10981 BIT_SHIFT_X I
+	FIELD field_10982 BITS_Z J
```

Comment:
Seems to be `BIT_MASK_CHUNK_SECTION` in 1.14-Snapshots according to yarn mappings.